README.md

```diff
@@ -10,9 +10,9 @@ This JavaScript library is part of the [Microsoft Teams developer platform](http
 ## Getting Started
 
 1. Clone the repo
-2. `yarn install` from repo root
-2. `yarn build` from repo root
-3. to run Unit test `yarn test`
+2. Run `yarn install` from repo root
+2. Run `yarn build` from repo root
```

Comment:
while you're here, can you fix the numbering?

Reply:
Done.